Automattic/wp-calypso
# Limit Global Styles: Added the Premium design tools to the signup and plan compare features

#### Proposed Changes

* We now list the `Premium design tools` in the plan comparison screen.

#### Testing Instructions



* Create a new site
* In the plan comparison screen you should see the `Premium design tools` feature for all paid plans.

#### Pre-merge Checklist






Related to 899-gh-Automattic/dotcom-forge
